### PR TITLE
 adding cluster name to subnetwork name

### PR DESCRIPTION
**Pull request by ezalejski**
in case of multiple clusters in the same workspace

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,27 @@
 # terraform-google-kubernetes-cluster
```
Comment (filiphaftek) [inline README.md]: Please change`terraform-google-kubernetes-cluster` -> `Overview.`

File: README.md [hunk 2]
```diff
@@ -1,8 +1,27 @@
...
 GKE Kubernetes module with node pools submodule
 
+Table of contents
+=================
+
+   * [terraform-google-kubernetes-cluster](#terraform-google-kubernetes-cluster)
+      * [1. Features](#1-features)
+      * [2. Usage:](#2-usage)
```
Comment (filiphaftek) [inline README.md]: Please remove `:`

File: README.md [hunk 3]
```diff
@@ -1,8 +1,27 @@
...
+         * [a) cluster with default node pool on preemptible](#a-cluster-with-default-node-pool-on-preemptible)
```
Comment (filiphaftek) [inline README.md]: My proposal is to remove `a, b, c` and just leave unordered list, how about that?

File: README.md [hunk 4]
```diff
@@ -1,8 +1,27 @@
...
+         * [b) cluster with explicit definition of node pools (optional)](#b-cluster-with-explicit-definition-of-node-pools-optional)
+         * [c) multiple clusters:](#c-multiple-clusters)
```
Comment (filiphaftek) [inline README.md]: Please remove `:`

File: README.md (hunk 3 above)
Comment (ezalejski) [reply README.md]: 👍

**Pull request merged**